### PR TITLE
r/directory_service_service_directory - fix import + add attributes

### LINKED ISSUE #10716: import aws_directory_service_directory doesn't import connect_settings
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.13
+ provider.aws v2.34.0
+ provider.http v1.1.1
+ provider.null v2.1.2

### Affected Resource(s)

* aws_directory_service_directory

### Terraform Configuration Files

```hcl
resource "aws_directory_service_directory" "corp" {
  type = "ADConnector"
  size = "Small"
  name = "CORP.local"
  short_name = "CORP"
  description = "CORP"
  password = "secret"

  connect_settings {
    customer_username = "aws"
    customer_dns_ips = [
      "10.1.1.1",
      "10.2.1.1",
    ]
    subnet_ids = [
      "subnet-1",
      "subnet-2",
    ]
    vpc_id = "vpc-1"
  }

  tags = {}
}
```

### Expected Behavior

`terraform import` adds all attributes of the directory -- including `connect_settings` -- to the terraform state. Subsequent plans do not modify the AWS resource.

### Actual Behavior

After `import`, the `connect_settings` attribute is missing from the state, and subsequent plans _replace_ the resource. Output of `terraform state show`:

```hcl
resource "aws_directory_service_directory" "corp" {
    access_url        = "d-redacted.awsapps.com"
    alias             = "d-redacted"
    description       = "CORP"
    dns_ip_addresses  = [
        "10.3.1.1",
        "10.3.2.1",
    ]
    enable_sso        = false
    id                = "d-redacted"
    name              = "CORP.local"
    security_group_id = "sg-0"
    short_name        = "CORP"
    size              = "Small"
    tags              = {}
    type              = "ADConnector"
}
```

Output of `terraform plan`:

```txt
  # aws_directory_service_directory.corp must be replaced
-/+ resource "aws_directory_service_directory" "corp" {
      # ...

      + connect_settings { # forces replacement
          + customer_dns_ips  = [
              + "10.1.1.1",
              + "10.2.1.1",
            ] # forces replacement
          + customer_username = "aws" # forces replacement
          + subnet_ids        = [
              + "subnet-1",
              + "subnet-2",
            ] # forces replacement
          + vpc_id            = "vpc-1" # forces replacement
        }
    }
```

### Steps to Reproduce

1. Create an AWS Directory Service AD Connector manually in the AWS console.
1. Write a terraform configuration that describes the resource e.g. as above.
1. `terraform import aws_directory_service_directory.corp d-redacted`
1. `terraform plan`

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes #10716

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
resource_aws_directory_service_service_directory: add `availability_zones` attribute to `vpc_settings` block.
resource_aws_directory_service_service_directory: add `availability_zones` attribute to `connect_settings` block.
resource_aws_directory_service_service_directory: add plan time validation to `customer_dns_ips` in `connect_settings` block.
```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSDirectoryServiceDirectory_'
--- PASS: TestAccAWSDirectoryServiceDirectory_disappears (814.29s)
--- PASS: TestAccAWSDirectoryServiceDirectory_basic (596.66s)
--- PASS: TestAccAWSDirectoryServiceDirectory_microsoft (1876.52s)
--- PASS: TestAccAWSDirectoryServiceDirectory_microsoftStandard (1694.06s)
--- PASS: TestAccAWSDirectoryServiceDirectory_connector (1092.28s)
--- PASS: TestAccAWSDirectoryServiceDirectory_withAliasAndSso (837.93s)
--- PASS: TestAccAWSDirectoryServiceDirectory_tags (803.25s)
```

### CODE REVIEW

File: aws/structure.go [hunk 1]
```diff
@@ -1684,11 +1685,11 @@ func flattenDSConnectSettings(
 
 	settings := make(map[string]interface{})
 
-	settings["customer_dns_ips"] = schema.NewSet(schema.HashString, flattenStringList(customerDnsIps))
-	settings["connect_ips"] = schema.NewSet(schema.HashString, flattenStringList(s.ConnectIps))
```
[the remark this answers is not in the document]
Comment (DrFaust92) [reply aws/structure.go:1688]: Np, should the other PR be merged first and then ill rebase and address the comment?

[the remark this answers is not in the document]
Comment (gdavison) [reply aws/structure.go:1688]: #13395 has been merged

[the remark this answers is not in the document]
Comment (DrFaust92) [reply aws/structure.go:1688]: Awesome, will rebase and test

**Comment by DrFaust92** [comment]
```
--- PASS: TestAccAWSDirectoryServiceDirectory_basic (613.38s)
--- PASS: TestAccAWSDirectoryServiceDirectory_tags (745.58s)
--- PASS: TestAccAWSDirectoryServiceDirectory_microsoft (1813.82s)
--- PASS: TestAccAWSDirectoryServiceDirectory_microsoftStandard (1784.81s)
--- PASS: TestAccAWSDirectoryServiceDirectory_withAliasAndSso (788.02s)
--- PASS: TestAccAWSDirectoryServiceDirectory_disappears (611.80s)
--- PASS: TestAccAWSDirectoryServiceDirectory_connector (1064.39s)
```

File: aws/structure.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrFaust92) [reply aws/structure.go:1740]: done

**Pull request merged**